### PR TITLE
#2383 - Horizonatal Autoscaler value change

### PR DESCRIPTION
**Pull request by guru-aot**
Horizonatal Autoscaler value change from average value to CPU utilization

### CODE REVIEW

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the additional tests and research, looks good 👍

File: devops/openshift/forms-deploy.yml [hunk 1]
```diff
@@ -175,8 +175,8 @@ objects:
             resource:
               name: cpu
               target:
-                type: AverageValue
-                averageValue: "0.8"
+                type: Utilization
+                averageUtilization: 50
```
Comment (dheepak-aot) [inline devops/openshift/forms-deploy.yml:179]: why is it 50 for form ?

Comment (guru-aot) [reply devops/openshift/forms-deploy.yml:179]: As part of the load testing for forms, the value is deried as 50%. This is similar to the .8 as mentioned previously.

**Review by ann-aot — APPROVED** [review]
👍

**Review by dheepak-aot — APPROVED** [review]
👍

**Review by andrepestana-aot — COMMENTED** [review]
👍

**Pull request merged**